include/circt/Dialect/RTL/RTLStructure.td

```diff
@@ -25,7 +25,9 @@ def RTLModuleOp : RTLOp<"module",
 
   let skipDefaultBuilders = 1;
   let builders = [
-    OpBuilder<(ins "StringAttr":$name, "ArrayRef<ModulePortInfo>":$ports)>
+    OpBuilder<(ins "StringAttr":$name, "ArrayRef<ModulePortInfo>":$ports, 
+                   CArg<"ArrayRef<NamedAttribute>",
+                        "ArrayRef<NamedAttribute>()">:$attributes)>
```

Comment:
I think you can change the default value to `{}` to make this a bit more concise (if you care).